ddtrace/debugging/_probe/poller.py

```diff
@@ -0,0 +1,81 @@
+from enum import Enum
+import time
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import Optional
+
+from ddtrace.debugging._probe.model import Probe
+from ddtrace.debugging._remoteconfig import DebuggingRCV07
+from ddtrace.internal.logger import get_logger
+from ddtrace.internal.periodic import PeriodicService
+
+
+log = get_logger(__name__)
+
+DEFAULT_PROBE_POLLER_INTERVAL = 5
+DEFAULT_PROBE_STATUS_UPDATE_INTERVAL = 45 * 60  # 45 minutes
+
+
+class ProbePollerEvent(Enum):
```

Comment:
nit

```suggestion
class ProbePollerEvent(object):
```

I think we're moving away from using Enums do the performance impact. Tahir ran this benchmark a few weeks ago. The cost is minor so I'm not sure if it's worth optimizing in the profiler

```
Enum
$ python -m timeit -s 'from ddtrace.internal.sampling import SamplingMechanism' -c '_ = "%d" % SamplingMechanism.DEFAULT.value'
1000000 loops, best of 5: 240 nsec per loop

Simple Object
$ python -m timeit -s 'from ddtrace.internal.sampling import SamplingMechanismO as SamplingMechanism' -c '_ = "%d" % SamplingMechanism.DEFAULT'
5000000 loops, best of 5: 66.4 nsec per loop
```

Reply:
Good point!